css/style.css

```
@@ -16,4 +24,101 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+#nav-bar {position:fixed;
+          top:0; 
+          height: 70px;
+          width:100%;
+          right:40px;
+          background-color: white;
+                   }
 
+    #nav-bar ul {list-style-type:none; 
+                display:flex; 
+                justify-content: flex-end;
+                }     
+  
+    #nav-bar ul a:hover {color:#F15B2A;
+                        }
+
+    #nav-bar img {height:60px; 
+                  width:200px; position: absolute;
+                  left: 10px;
+                  top: 10px;
+                  z-index: -1;
+                 }
+
+    .nav-link {text-decoration:none; 
+              color:darkgray; 
+              font-weight:bolder; 
+              display:block; 
+              padding:8px; 
+             }
+    
+   .nav li:first-child > a {color: rgb(46, 44, 44);}
+
+
+article {margin-top:70px;}
+
+
+   h1 {color: white; font-weight:lighter; font-size: 2em;}
+   h2 {color: white; font-weight:lighter;}
```

```suggestion
   #intro h2 {color: white; font-weight:lighter;}
```
We want the `h2` to only be white in the `#intro` section, so we can update the selector to only apply to the `h2`s in the intro